src/internal/operators/shareReplay.ts

```diff
@@ -143,9 +143,10 @@ export function shareReplay<T>(bufferSize?: number, windowTime?: number, schedul
  * @see {@link share}
  * @see {@link publishReplay}
  *
- * @param {Number} [bufferSize=Infinity] Maximum element count of the replay buffer.
```

Comment:
Docs builder is smart enough to understand the default value and the type, so declaring them is not necessary:

![image](https://user-images.githubusercontent.com/28087049/187615741-790c53d1-c344-4798-9563-d53b20491fb2.png)